src/main.js

```diff
@@ -0,0 +1,159 @@
+import React from 'react';
+import { render as reactDomRender } from 'react-dom'; 
+import superagent from 'superagent';
+import './style/main.scss';
+
+const apiUrl = 'http://www.reddit.com/r';
+// /${searchFormBoard}.json?limit=${searchFormLimit}
+
+class SearchForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      redditBoard: '',
+      limit: '',
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleLimitChange = this.handleLimitChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({ redditBoard: event.target.value });
+  }
+
+  handleLimitChange(event) {
+    this.setState({ limit: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    this.props.redditBoardSelect(this.state.redditBoard);
```

Comment:
change `redditBoardSelect` to take in two arguments, the `redditBoard` and `limit`